perfTest/run.sh

```diff
@@ -1,24 +1,30 @@
 #!/usr/bin/env bash
```

Comment:
This script should still run locally as before, although I haven't been able to test that thoroughly.